project/Frontend.scala

```diff
@@ -157,9 +157,11 @@ object Frontend extends Build with Prototypes {
     )
 
   val faciaEndToEnd = application("facia-end-to-end")
-    .dependsOn(commonWithTests)
-    .dependsOn(facia, faciaTool, faciaPress)
-    .aggregate(common, facia, faciaTool, faciaPress)
+    .dependsOn(faciaTool, facia, faciaPress)
+    .aggregate(faciaTool, facia, faciaPress)
+    .settings(
+      javaOptions in Runtime += "-Dconfig.file=facia-end-to-end/conf/facia-end-to-end.application.conf"
```

Comment:
I believe it does a conf merge, thanks to the typesafe conf library. Given how similar the conf files are, this is usually benign, I just did it for completeness. I want us to move towards Play servers which only have a single router (`routes`), a single conf (`application.conf`), and a single `controllers.Assets` object